WebContent/js/controllers/FilterController.js

```diff
@@ -0,0 +1,90 @@
+mobileStoreApp.controller('FilterController', ['$scope', function($scope){
```

Comment:
ts odd to comment all statements. Need not comment for self explanatory vars/ functions.